run-pylint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Just minor thing: some sections of this bash file are not correctly indented. Can you pls indent correctly? Just for readability.

Reply:
Rebased and fixed the indentation.